### ISSUE TITLE
opencv_version

### ISSUE THREAD

**Issue by SomnusHe** [ultravideo/Open3DGen#5]
hello， I would like to know which version of opencv you are using？

**Comment by teo3n** [comment]
I don't think it matters at all as far as you use any major version 4.x onwards. Just use the latest one (4.5.5 right now) and you will be fine.

**Comment by AiYoWeiYL** [comment]
Hi, may I ask which version of Eigen is? I met the following problems when compiling (using OpenCV 4.5.5 , Open3D 0.10.1)

In file included from /usr/local/include/eigen3/unsupported/Eigen/CXX11/src/util/CXX11Meta.h:14:0,
                 from /usr/local/include/eigen3/unsupported/Eigen/CXX11/Tensor:31,
                 from /home/yr/cvlib/opencv-4.5.5/modules/core/include/opencv2/core/eigen.hpp:64,
                 from /home/yr/Open3DGen/src/math_modules.h:9,
                 from /home/yr/Open3DGen/src/math_modules.cpp:1:
/usr/local/include/eigen3/unsupported/Eigen/CXX11/src/util/EmulateArray.h:254:30: error: redeclared with 1 template parameter
 template <typename T> struct array_size;
                              ^~~~~~~~~~
In file included from /usr/local/include/Open3D/3rdparty/Eigen/Core:151:0,
                 from /usr/local/include/Open3D/Camera/PinholeCameraIntrinsic.h:29,
                 from /usr/local/include/Open3D/Open3D.h:30,
                 from /home/yr/Open3DGen/src/math_modules.h:2,
                 from /home/yr/Open3DGen/src/math_modules.cpp:1:
/usr/local/include/Open3D/3rdparty/Eigen/src/Core/util/Meta.h:402:55: note: previous declaration ‘template<class T, class EnableIf> struct Eigen::internal::array_size’ used 2 template parameters
 template<typename T, typename EnableIf = void> struct array_size {
                                                       ^~~~~~~~~~
In file included from /usr/local/include/eigen3/unsupported/Eigen/CXX11/src/util/CXX11Meta.h:14:0,
                 from /usr/local/include/eigen3/unsupported/Eigen/CXX11/Tensor:31,
                 from /home/yr/cvlib/opencv-4.5.5/modules/core/include/opencv2/core/eigen.hpp:64,
                 from /home/yr/Open3DGen/src/math_modules.h:9,
                 from /home/yr/Open3DGen/src/math_modules.cpp:1:
/usr/local/include/eigen3/unsupported/Eigen/CXX11/src/util/EmulateArray.h:255:41: error: redefinition of ‘struct Eigen::internal::array_size<const std::array<_Tp, _Nm> >’
 template<class T, std::size_t N> struct array_size<const std::array<T,N> > {
